content/cli/v7/using-npm/registry.md

```diff
@@ -20,7 +20,7 @@ package info.
 
 npm is configured to use the **npm public registry** at
 <https://registry.npmjs.org> by default. Use of the npm public registry is
-subject to terms of use available at <https://www.npmjs.com/policies/terms>.
+subject to terms of use available at <https://docs.npmjs.com/policies/terms>.
```

Comment:
Should be updated in https://github.com/npm/cli/blob/latest/docs/content/using-npm/registry.md